### ISSUE TITLE
Crash in ErrorsKt.java line 53, com.qonversion.android.sdk.ErrorsKt.toQonversionError

### ISSUE THREAD

**Issue by krottv** [qonversion/android-sdk#71]
I have qonversion version 2.0.2.

Some of users experience crashes with following stacktrace:

```
Caused by org.json.JSONException
Value <html> of type java.lang.String cannot be converted to JSONObject
org.json.JSON.typeMismatch (JSON.java:112)
org.json.JSONObject.<init> (JSONObject.java:176)
com.qonversion.android.sdk.ErrorsKt.toQonversionError (ErrorsKt.java:53)
com.qonversion.android.sdk.QonversionRepository$initRequest$1$1.invoke (QonversionRepository.java:299)
com.qonversion.android.sdk.QonversionRepository$initRequest$1$1.invoke (QonversionRepository.java:25)
com.qonversion.android.sdk.CallBackKt.onResponse (CallBackKt.java:24)
retrofit2.ExecutorCallAdapterFactory$ExecutorCallbackCall$1$1.run (ExecutorCallAdapterFactory.java:70)
```

Android versions: 9, 10

**Comment by Maria-Bordunova** [comment]
Hi, @krottv .
Thank you for contacting us. We will investigate the problem and return with the answer in the near future.

**Comment by Maria-Bordunova** [comment]
Could you provide us a bit more information like Qonversion project name, project key, client uid. Anything you have would be helpful. You can contact me by email maria@qonversion.io

**Comment by Maria-Bordunova** [comment]
Thank you for waiting, @krottv
We finally figured it out. We will fix it soon. I'll let you know when it's ready.

**Comment by krottv** [comment]
@Maria-Bordunova Great. Thanks!

**Comment by Maria-Bordunova** [comment]
@krottv ,we published new release SDK version [2.1.1](https://github.com/qonversion/android-sdk/releases/tag/2.1.1) .
It's already available for updating ✅.
Thank you again for the report and contribution .
